docs/internal/investigations/dapr-remote-runtime-configuration/configuration_concept.md

```diff
@@ -0,0 +1,29 @@
+# Remote runtime management concept
+
+## Overview
+
+This document describes the concept of runtime management with the usage of [Dapr](https://dapr.io).
+
+## Architecture
+
+![Concept image](assets/concept.png?raw=true "Concept")
+
+The diagram above shows the future architecture of Kyma environment.
+There are two central places, Compass and Kyma Control Plane along with runtimes which will communicate with them.
+
+### Compass
+It is responsible for connecting applications to runtimes, along with controlling and monitoring them.
+The System Broker will be responsible for fetching data from different types of runtimes (e.g. Cloud Foundry or customer Kubernetes cluster with service catalog installed on it).  
+
+### Kyma Control Plane
+The Kyma Control Plane is responsible for managing the Kyma instances. It allows to provision Kyma Runtimes as well as it is a central place where the Kyma instances can fetch their configuration from. 
+The Runtime Director exposes the API for the Kyma Runtimes configuration. 
+
+### Runtimes
+The diagram shows possible runtimes, which will be supported. It includes Cloud Foundry runtime, Customer own cluster with Service Catalog installed, Kyma Managed Runtime, and Kyma Standalone Runtime. 
+Each supported Kyma instance has a dedicated agent component that is responsible for handling the communication with the Kyma Central Plane. The agent component fetches the configuration from the Runtime Director and applies it to the desired environment.
+
+## Kyma Runtime configuration from the Control Plane
+The main idea is to have a component in Kyma Control Plane (Runtime Director on image), which would be a point of communication with Runtimes through Agents to manage them. It would allow also to configure them from one central point.
```

Comment:
```suggestion
The main idea is to have a component in Kyma Control Plane that will be a point of communication for Agents to manage the Runtimes. This component would be the image of Runtime Director. It would also allow for configuring Runtimes from one central place.
```